sql/modules/Roles.sql

```diff
@@ -326,7 +382,14 @@ SELECT lsmb__grant_perms(role, 'file_incoming', 'DELETE'),
                     'file_attach_part', 'file_attach_eca']) role;
 
 \echo Contact Management
-SELECT lsmb__create_role('contact_read');
+SELECT lsmb__create_role('contact_read',
+                         $DOC$
+                         This role allows searching and viewing entities, persons and companies (contacts).
+
+                         Combine this role with one or more 'contact_class_*' roles to be able to access
+                         contacts of the specific class. By itself, this role does not provide sufficient rights.
```

Comment:
Is this similar to the `base_user` that a user with `contact_read` don't really get any access from this role?

I'm not sure what the role actually permits, if I can search and view `entities, persons and companies` for what permission do I need the `contact_class_*`? Does the `contact_read` only allow the user to see a list of search results perhaps?

Reply:
The roles are complementary: you need both "contact_class_vendors" and "contact_read" in order to be able to read contact data of type "vendor" (but not be able to access "customer" data).

Reply:
Would a better wording be something like:

> When paired with one or more `contact_class_*` role/-s, this role allows searching and viewing entities, persons and companies (contacts)
> 
> Each `contact_class_<resource>` role, when paired with `contact_read`, enables this access for the specific `<resoucre>`. On it's own, the `contact_read`-role does not provide any rights.

or similar?

Reply:
Done.